profile/profile.go

```diff
@@ -301,18 +298,6 @@ func (profile *Profile) addEndpointyEntry(cfgKey, newEntry string) {
 		}
 	}()
 
-	// When finished increase the revision counter of the layered profile.
-	defer func() {
-		if !changed || profile.layeredProfile == nil {
-			return
-		}
-
-		profile.layeredProfile.Lock()
-		defer profile.layeredProfile.Unlock()
-
-		profile.layeredProfile.RevisionCounter++
-	}()
-
```

Comment:
How does the revision counter get handled now?

Reply:
Profiles are marked as outdated when they are saved. The next operation that uses the profile checks for outdated profiles, reloads them and increases the counter. Only having a single way of doing that makes this simpler and less error prone.